packages/webpack/docs_test.js

```diff
@@ -0,0 +1,6 @@
+const actual = require('fs').readFileSync(
```

Comment:
but there aren't actually any markdown files in the PR, seems like the docs feature isn't complete.

Not sure if we should finish it, or drop it for now and add when it gets more mature

Reply:
not sure why this is here also. perhaps @mrmeku knows something